docs/examples/kafka-sklearn/README.md

```diff
@@ -0,0 +1,248 @@
+
+# We learn the digits on the first half of the digits
+classifier.fit(X_train, y_train)
+```
+
+
+
+
+    SVC(gamma=0.001)
+
+
+
+### Saving our trained model
+
+To save our trained model, we will serialise it using `joblib`.
+While this is not a perfect approach, it's currently the recommended method to persist models to disk in the [`scikit-learn` documentation](https://scikit-learn.org/stable/modules/model_persistence.html).
+
+Our model will be persisted as a file named `mnist-svm.joblib`
+
+
+```python
+import joblib
+
+model_file_name = "mnist-svm.joblib"
+joblib.dump(classifier, model_file_name)
+```
+
+
+
+
+    ['mnist-svm.joblib']
+
+
+
+## Serving
+
+Now that we have trained and saved our model, the next step will be to serve it using `mlserver`. 
+For that, we will need to create 2 configuration files: 
+
+- `settings.json`: holds the configuration of our server (e.g. ports, log level, etc.).
+- `model-settings.json`: holds the configuration of our model (e.g. input type, runtime to use, etc.).
+
+### `settings.json`
+
+
+```python
+%%writefile settings.json
+{
+    "debug": "true",
+    "kafka_enable": "true"
+}
+```
+
+    Overwriting settings.json
+
+
+### `model-settings.json`
+
+
+```python
+%%writefile model-settings.json
+{
+    "name": "mnist-svm",
+    "implementation": "mlserver_sklearn.SKLearnModel",
+    "parameters": {
+        "uri": "./mnist-svm.joblib",
+        "version": "v0.1.0"
+    }
+}
+```
+
+    Overwriting model-settings.json
+
+
+### Start serving our model
+
+Now that we have our config in-place, we can start the server by running `mlserver start .`. This needs to either be ran from the same directory where our config files are or pointing to the folder where they are.
+
+```shell
+mlserver start .
+```
+
+Since this command will start the server and block the terminal, waiting for requests, this will need to be ran in the background on a separate terminal.
+
+### Send test inference request
+
+We now have our model being served by `mlserver`.
+To make sure that everything is working as expected, let's send a request from our test set.
+
+For that, we can use the Python types that `mlserver` provides out of box, or we can build our request manually.
+
+
+```python
+import requests
+
+x_0 = X_test[0:1]
+inference_request = {
+    "inputs": [
+        {
+          "name": "predict",
+          "shape": x_0.shape,
+          "datatype": "FP32",
+          "data": x_0.tolist()
+        }
+    ]
+}
+
+endpoint = "http://localhost:8080/v2/models/mnist-svm/versions/v0.1.0/infer"
+response = requests.post(endpoint, json=inference_request)
+
+response.json()
+```
+
+
+
+
+    {'model_name': 'mnist-svm',
+     'model_version': 'v0.1.0',
+     'id': '53e3c2d5-456c-41a6-9e7f-6ba0ef717c3c',
+     'parameters': None,
+     'outputs': [{'name': 'predict',
+       'shape': [1],
+       'datatype': 'INT64',
+       'parameters': None,
+       'data': [8]}]}
+
+
+
+
+```python
```

Comment:
Should we add a few words here about how we are sending / returning payloads to / from Kafka?